packages-exp/auth-exp/src/core/user/additional_user_info.test.ts

```diff
@@ -35,13 +31,13 @@ import { User, UserCredential } from '../../model/user';
 import { testAuth, testUser } from '../../../test/helpers/mock_auth';
 
 describe('core/user/additional_user_info', () => {
-  const userProfileWithLogin: UserProfile = {
+  const userProfileWithLogin: Record<string, unknown> = {
```

Comment:
is this type necessary here?

Reply:
Yes; object literals should have a strong type specified